packages/echo-core/rollup.config.js

```diff
@@ -1,10 +1,10 @@
 import babel from '@rollup/plugin-babel';
 import commonjs from '@rollup/plugin-commonjs';
 import { nodeResolve } from '@rollup/plugin-node-resolve';
+import typescript from '@rollup/plugin-typescript';
```

Comment:
Changing the `rollup-plugin-typescript2` to the official typescript rollup plugin.